code/parse/sexp.h

```diff
@@ -734,7 +734,6 @@ class waypoint_list;
 #define OP_GET_ETS_VALUE					(0x0022 | OP_CATEGORY_CHANGE2 | OP_NONCAMPAIGN_FLAG)	// niffiwan
 #define OP_SET_ETS_VALUES					(0x0023 | OP_CATEGORY_CHANGE2 | OP_NONCAMPAIGN_FLAG)	// niffiwan
 #define OP_CALL_SSM_STRIKE					(0x0024 | OP_CATEGORY_CHANGE2 | OP_NONCAMPAIGN_FLAG) // X3N0-Life-Form
-#define OP_SET_MOTION_DEBRIS				(0x0025 | OP_CATEGORY_CHANGE2 | OP_NONCAMPAIGN_FLAG)    // The E
```

Comment:
I recommend renumbering the subsequent sexps, or moving the `OP_CHANGE_BACKGROUND` definition, so that we don't leave any gaps in the operator numbering sequence.

Reply:
Good point, fixed. The dynamic SEXP system is also not really set up to handle holes in the operators so this is also a technical necessity.